subsys/net/lib/tftp/Kconfig

```diff
@@ -0,0 +1,48 @@
+# Socket TFTP Library for Zephyr
+
+# Copyright (c) 2020 InnBlue
+# SPDX-License-Identifier: Apache-2.0
+
+config TFTP_LIB
+	bool "Socket TFTP Library Support"
```

Comment:
Could you mark this as experimental, so just add `... [EXPERIMENTAL]` at the end of this text. This way we can more easily change the API when needed.